packages/paste-website/src/pages/index.tsx

```diff
@@ -54,7 +55,13 @@ const IndexPage: React.FC<{}> = (): React.ReactElement => {
             </Button>
           </Box>
           <Box display="inline" marginRight="space40">
-            <Button as="a" href="/components" size="default" variant="secondary">
+            <Button
+              size="default"
+              variant="secondary"
+              onClick={() => {
```

Comment:
We need the same treatment on the other buttons above and below this one.

Reply:
Yup. Just pushed the other two fixes.